<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FemGui::DlgSettingsFemCodeAsterImp</class>
 <widget class="QWidget" name="FemGui::DlgSettingsFemCodeAsterImp">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>408</width>
    <height>145</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CodeAster</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QGroupBox" name="gb_codeaster_param">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="title">
      <string>Code Aster binary</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="1">
         <widget class="Gui::PrefCheckBox" name="cb_codeaster_binary_std">
          <property name="text">
           <string>Search in known binary directories</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
          <property name="prefEntry" stdset="0">
           <cstring>UseStandardCodeAsterLocation</cstring>
          </property>
          <property name="prefPath" stdset="0">
           <cstring>Mod/Fem/CodeAster</cstring>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="l_codeaster_binary_path">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>CA binary path</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="Gui::PrefFileChooser" name="fc_codeaster_binary_path" native="true">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="sizeIncrement">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="baseSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="toolTip">
           <string>Leave blank to use default Code Aster binary file location</string>
          </property>
          <property name="prefEntry" stdset="0">
           <cstring>codeasterBinaryPath</cstring>
          </property>
          <property name="prefPath" stdset="0">
           <cstring>Mod/Fem/CodeAster</cstring>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="l_codeaster_write_comments">
          <property name="text">
           <string>Comments</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="Gui::PrefCheckBox" name="cb_codeaster_write_comments">
          <property name="text">
           <string>Write comments to input file</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
          <property name="prefEntry" stdset="0">
           <cstring>writeCommentsToInputFile</cstring>
          </property>
          <property name="prefPath" stdset="0">
           <cstring>Mod/Fem/CodeAster</cstring>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>Gui::FileChooser</class>
   <extends>QWidget</extends>
   <header>Gui/FileDialog.h</header>
  </customwidget>
  <customwidget>
   <class>Gui::PrefFileChooser</class>
   <extends>Gui::FileChooser</extends>
   <header>Gui/PrefWidgets.h</header>
  </customwidget>
  <customwidget>
   <class>Gui::PrefCheckBox</class>
   <extends>QCheckBox</extends>
   <header>Gui/PrefWidgets.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="Resources/Fem.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>cb_codeaster_binary_std</sender>
   <signal>toggled(bool)</signal>
   <receiver>l_codeaster_binary_path</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>406</x>
     <y>45</y>
    </hint>
    <hint type="destinationlabel">
     <x>148</x>
     <y>68</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cb_codeaster_binary_std</sender>
   <signal>toggled(bool)</signal>
   <receiver>fc_codeaster_binary_path</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>406</x>
     <y>45</y>
    </hint>
    <hint type="destinationlabel">
     <x>406</x>
     <y>68</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
